starter_scripts/exaslct_within_docker_container.sh

```diff
@@ -11,12 +16,56 @@ else
   terminal_parameter=""
 fi
 
+
+relevant_mount_point_arguments=("--flavor-path" "--export-path" "--output-directory" "--temporary-base-directory"
+                                "--cache-directory" "--save-directory" "--task-dependencies-dot-file" "--test-folder"
+                                "--test-file" "--temporary-base-directory")
+
+function _get_mount_point_argument() {
```

Comment:
you still need to know what type of argument you handle, because InFile and InDirectory, or OutFile and OutDirectory need different handling. For InFile and OutFile it is probably the best to mount their parent directory. But, if the parent directory for InFile doesn't exist we fail. We also fail for InDirectory if the directory doesn't exist. For OutDirectory, we can instead safely create the directory if not already exists.